Tribler/Core/Modules/versioncheck_manager.py

```diff
@@ -0,0 +1,39 @@
+from distutils.version import LooseVersion
+import json
+import logging
+from twisted.web.client import Agent, readBody
+from twisted.web.http_headers import Headers
+from Tribler.Core.Utilities.twisted_thread import reactor
+from Tribler.Core.simpledefs import NTFY_INSERT, NTFY_NEW_VERSION
+from Tribler.Core.version import version_id
+from Tribler.dispersy.taskmanager import TaskManager
+
+
+VERSION_CHECK_URL = 'https://api.github.com/repos/tribler/tribler/releases/latest'
+
+
+class VersionCheckManager(TaskManager):
+
+    def __init__(self, session):
+        super(VersionCheckManager, self).__init__()
+
+        self._logger = logging.getLogger(self.__class__.__name__)
+        self.session = session
+
+    def schedule_version_check(self, delay):
+        self.register_task("tribler version check", reactor.callLater(delay, self.check_new_version))
+
+    def check_new_version(self):
+
+        def parse_body(body):
+            version = json.loads(body)['name'][1:]
```

Comment:
@whirm the API endpoint does always give the latest full stable release according to the Github API docs.